lib/freeze.js

```diff
@@ -40,6 +40,8 @@ const contentTypes = {
     '.woff2': 'application/font-woff2'
 };
 
+const inlineTypes = [':base64:', ':encodeURI:', ':encodeURIComponent:', ':encodeURIComponentSvg:'];
```

Comment:
it's better to switch to hash to make O(1) lookup.
and let's stick to `var` for consistency reason.